MOBILEDOC.md

```diff
@@ -96,6 +96,29 @@ the start of the `value` text. As these tags are opened, then create a stack
 of opened markups. The `numberOfClosedMarkups` says how many markups should
 be closed at the end of a `value`.
 
+Text sections may contain [ATOMS](ATOMS.md) and markups. Atoms are self closing,
+and so the `numberOfClosedMarkups` will always be 0. Atoms have an additional
+`payload` entry in the array:
+
+```
+{
+  version: "0.3",
+  sections: [[
+    ["b"],                                ──── Bold Markup at index 0
+    ["i"],                                ──── Italic Markup at index 1
+    ["atom", "mention"]                   ──── Mention Atom at index 2
+  ],[
+    [openMarkupsIndexes, numberOfClosedMarkups, value, payload],
+    [[2], 0, "Bob", { id: 42 }],          ──── Mention Atom with payload
+    [[1], 0, "Example opening i tag"],
```

Comment:
I've wanted to separate sections from markups for a while now, so I am +1 to the above.

The `sections` you have above are not real sections, though. I think you meant to wrap in a section, so it should be:
```
sections: [1, "P", [                                  ——— a "p" markup section containing two markers:
  [0, [2], 0, 'bob'],                                    ─ ─ ─ marker type 0 (normal marker), italic
  [1, 0, 'martin', { url: 'google.com' } ]          ─ ─ ─ marker type 1 (atom), atom index 0 ('mention'), text, payload
]]
```